docs/README.md

```diff
@@ -56,10 +63,31 @@ The second annotation type is a **comment**. Comments are text annotations, just
 Comments can also be *Upvoted*, *Downvoted* or *Removed* by other users. 
 
 ### 3. GEOTAGGING
-Lastly, **geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be *Upvoted*, *Downvoted* or *Removed* by other users.
+**Geotagging** is a special kind of tag that refers specifically to a location. A user can start typing the location name in the input field under the map, and then select the appropriate location from the suggested ones on the drop-down below. Geotags can also be validated by the supported feedback methods (upvoting or percentage rating).
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image9.jpg" width="auto">
 
+### 4. COLORTAGGING
+
+**Colortagging** is a special kind of tag that refers to color. A user can select the color from a predefined color pallette. Colortags can also be validated by the supported validation methods (upvoting or percentage rating).
+
+<img src="_media/annotating-colortagging.png" width="auto">
+
+### 5. IMAGETAGGING
+
+**IMAGETAGGING** is a special kind of tag that refers to tagging images. A user can see the original image along with algorithm-generated ones (e.g. by Super-Resolution models). The user can then click *Compare* to start the comparison.
```

Comment:
Verify this. 

BTW Is it only Super-Resolution models?

Reply:
Spiros ->
Image tagging only works with base annotations

Suggestion ->
"refers to tagging images by making use of the base annotation functionality"